rotkehlchen/exchanges/kraken.py

```diff
@@ -1146,7 +1061,23 @@ def history_event_from_kraken(
 
                 # Make sure to not generate an event for KFEES that is not of type FEE
                 if asset != A_KFEE:
-                    event = HistoryEvent(
+                    # Process asset movements
+                    if event_type in {HistoryEventType.DEPOSIT, HistoryEventType.WITHDRAWAL}:
+                        group_events.extend(
+                            (idx, event) for event in create_asset_movement_with_fee(
+                                timestamp=timestamp,
+                                location=Location.KRAKEN,
+                                event_type=event_type,  # type: ignore  # will be deposit or withdrawal
```

Comment:
I've checked and type casting seems to be the only way to avoid this so the ignore is okey